Add consensus CNV calling in method section (#24)
d3b-center/OpenPedCan-methods: content/06.methods.md

##### Consensus CNV Calling
<!-- TODO: needs update -->

We adopted the consensus CNV calling described in OpenPBTA manuscript [doi:10.1016/j.xgen.2023.100340] with minor adjustments. 

> For each caller and sample, we called CNVs based on consensus among Control-FREEC ([@https://doi.org/10.1093/bioinformatics/btq635]; [@https://doi.org/10.1093/bioinformatics/btr670]), CNVkit ([@doi: 10.1371/journal.pcbi.1004873]), and GATK ([@doi:

Reviewer: ```suggestion
For each caller and sample with WGS performed, we called CNVs based on consensus among Control-FREEC ([@https://doi.org/10.1093/bioinformatics/btq635]; [@https://doi.org/10.1093/bioinformatics/btr670]), CNVkit ([@doi: 10.1371/journal.pcbi.1004873]), and GATK ([@doi:
```

10.1101/gr.107524.110]). 
Sample and consensus caller files with more than 2,500 CNVs were removed to de-noise and increase data quality, based on cutoffs used in GISTIC ([@https://doi.org/10.1186/gb-2011-12-4-r41]). 

> For each sample, we included the regions in the final consensus set: 1) regions with reciprocal overlap of 50% or more between at least two of the callers; 2) smaller CNV regions in which more than 90% of regions are covered by another caller. 

Reviewer: ```suggestion
For each sample, we included the following regions in the final consensus set: 1) regions with reciprocal overlap of 50% or more between at least two of the callers; 2) smaller CNV regions in which more than 90% of regions were covered by another caller. 
```

> For GATK, if a panel of normal was not provided, the correlated sample would be skipped. 

Reviewer: ```suggestion
For GATK, if a panel of normal was not able to be created (required 30 male and 30 female with the same sequencing platform), consensus was not run for tumors with WGS performed on that sequencing platform. 
```

> All the copy number alteration included in the consensus file were shown in algorithm two algorithms. 

Reviewer: ```suggestion
```

We defined copy number as NA for any regions that had a neutral call for the samples included in the consensus file. 
We merged CNV regions within 10,000 bp of each other with the same direction of gain or loss into single region. 

> Any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions, or that were shorter than 3000 bp were filtered out.

Reviewer: ```suggestion
Any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions, or that were shorter than 3000 bp were filtered out.
The CNVKit calls for WXS samples were appended to the consensus CNV file.
```
